examples/powershell/create-and-monitor-export-job.ps1

```diff
@@ -0,0 +1,103 @@
+###################################
+# create-and-monitor-export-job.ps1
+###################################
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+function Get-Bearer-Token {
+  $headers = New-Object "System.Collections.Generic.Dictionary[[String],[String]]"
+  $headers.Add("Authorization", "Basic $AUTH")
+  $headers.Add("Accept", "application/json")
+  $headers.Add("Content-Type", "application/x-www-form-urlencoded")
+  $response = Invoke-RestMethod "$OKTA_URI_WITH_PARAMS" -Method "POST" -Headers $headers -Body $body
+  Write-Host '---------------------------------------------------------------------------------------------------------------------'
+  Write-Host 'The latest bearer token used to authenticate with the AB2D API (expires in 1 hour)'
+  Write-Host '---------------------------------------------------------------------------------------------------------------------'
+  Write-Host $response.access_token
+  Write-Host ''
+  return $response.access_token
+}
+
+# Set API variables
+
+$AUTH = "$BASE64_ENCODED_ID_PASSWORD"
+$OKTA_URI_WITH_PARAMS = "$AUTHENTICATION_URL`?grant_type=client_credentials&scope=clientCreds"
+$EXPORT_URL = "$AB2D_API_URL/v1/fhir/Patient/`$export`?_outputFormat=application%2Ffhir%2Bndjson&_type=ExplanationOfBenefit"
+Write-Host ''
+Write-Host '---------------------------------------------------------------------------------------------------------------------'
+Write-Host 'The AUTH used for getting bearer token'
+Write-Host '---------------------------------------------------------------------------------------------------------------------'
+Write-Host $AUTH
+Write-Host ''
+Write-Host '---------------------------------------------------------------------------------------------------------------------'
+Write-Host 'The OKTA URI used for getting bearer token'
+Write-Host '---------------------------------------------------------------------------------------------------------------------'
+Write-Host $OKTA_URI_WITH_PARAMS
+Write-Host ''
+Write-Host '---------------------------------------------------------------------------------------------------------------------'
+Write-Host 'The AB2D API endpoint for starting an export job'
+Write-Host '---------------------------------------------------------------------------------------------------------------------'
+Write-Host $EXPORT_URL
+Write-Host ''
+
+# Get initial bearer token
+
+$BEARER_TOKEN = Get-Bearer-Token
+
+# Create an export job
+
+$headers = New-Object "System.Collections.Generic.Dictionary[[String],[String]]"
+$headers.Add("Accept", "application/json")
+$headers.Add("Prefer", "respond-async")
+$headers.Add("Authorization", "Bearer $BEARER_TOKEN")
+$response = Invoke-WebRequest "$EXPORT_URL" -Method 'GET' -Headers $headers -Body $body
```

Comment:
Do you provide/save a job id in case monitoring fails?

Reply:
Examples have been moved to different repos.